ui/src/app/views/concept-homepage/component.html

```diff
@@ -48,16 +48,21 @@
   <ng-template #chartBox>
     <div class="chart-box lower-box">
       <div class="chart-selector-tabs">
-        <div *ngFor="let domain of conceptDomainList" class="chart-selector-tab">
-          <button (click)="selectDomain(domain.domain)" class="domain-selector-button btn btn-link nav-link nav-item" [class.active]="domain.domain === selectedDomain" [disabled]="searchLoading">
+        <div *ngFor="let domain of conceptDomainCounts" class="chart-selector-tab">
+          <button (click)="selectDomain(domain.domain)" class="domain-selector-button btn btn-link nav-link nav-item" [class.active]="domain.domain === selectedDomain" [disabled]="searchLoading || !completedDomainSearches.includes(domain.domain)">
             <div class="chart-selector-header">
               {{domain.name}}
             </div>
-            <!-- TODO (RW-1277): Make this the counts returned from search. -->
-            <div>{{domain.allConceptCount}}</div>
+            <div>{{domain.concept_count}}</div>
```

Comment:
Hmm, we should fix this - I suspect the underscore was an accident @danqrodney ?

Reply:
It was, I have already told Dan and he is fixing it. If he gets a fix in before this goes in, I will rebase and change, otherwise I shall send him the two lines to change.

Reply:
Yeah, I'm fixing this. I'll update the PR to also include the corresponding UI change.

Reply:
There's also a test file that will need to change. (But only when this goes in) I can send you a list of lines of code after this goes in.